data/mods/Aftershock/afs_professions.json

```diff
@@ -8,10 +8,20 @@
     "traits": [ "DEBUG_HS", "DEBUG_NODMG", "DEBUG_STORAGE", "DEBUG_BIONIC_POWER", "DEBUG_NOTEMP", "DEBUG_LS" ],
     "CBMs": [ "bio_power_storage_mkII", "bio_power_storage_mkII", "bio_power_storage_mkII", "bio_power_storage_mkII", "bio_tools" ],
     "skills": [ { "level": 8, "name": "fabrication" }, { "level": 8, "name": "mechanics" } ],
+    "items": { "both": [ "linuxtshirt", "pants_cargo", "socks", "sneakers" ], "male": [ "briefs" ], "female": [ "bra", "panties" ] }
+  },
+  {
+    "type": "profession",
+    "ident": "afs_skiier",
+    "name": "Skiier",
+    "description": "The slopes of New England are nice this time of year, and you were spending the month on vacation at a resort when you were evacuated.",
+    "points": 2,
+    "traits": [ "AFS_PROF_SKIIER" ],
+    "skills": [ { "level": 3, "name": "dodge" }, { "level": 2, "name": "survival" } ],
```

Comment:
Why the survival? It could be the most ordinary skiier in the world.

Reply:
This is implied to be an experienced skiier, due to their profession trait. The survival skill represents their knowledge of alpine conditions and precautions they might unconsciously take. Survival 1 isn't very high at all.